tests/components/mqtt/test_alarm_control_panel.py

```diff
@@ -264,20 +267,56 @@ async def test_ignore_update_state_if_unknown_via_state_topic(
             ),
             AlarmControlPanelEntityFeature.ARM_HOME
             | AlarmControlPanelEntityFeature.ARM_AWAY,
+            True,
+        ),
+        (
+            help_custom_config(
+                alarm_control_panel.DOMAIN,
+                DEFAULT_CONFIG,
+                ({"supported_features": "invalid"},),
+            ),
+            AlarmControlPanelEntityFeature.ARM_HOME
+            | AlarmControlPanelEntityFeature.ARM_AWAY,
```

Comment:
You are not using these fro the False case. So you can assign `None`

Reply:
Did you miss the other 2 occurrences?

Reply:
Yeah I did... So sorry about that, I got pretty tired at that time of the day and with so many back and forths. Thank you for catching this!